toplevel/coqargs.ml

```diff
@@ -322,6 +322,9 @@ let usage batch =
   then Usage.print_usage_coqc ()
   else Usage.print_usage_coqtop ()
 
+let deprecated_coqtop opt_name =
+  Feedback.msg_warning Pp.(seq [str "The option "; str opt_name; str" is deprecated, please use coqc"])
```

Comment:
Why isn't this using cwarnings? Then we could just pass `-w -deprecate-compile-arg` before -compile in coqc.

Reply:
I didn't use `cwarnings` as in a sense, this warning should not be allowed to be disabled; but indeed no problem in doing what you propose.